### ISSUE TITLE
(Focus, Tag, Date)Pickers return <span> tags; should be <button>

### ISSUE THREAD

**Issue by ptwu** [cornell-dti/samwise#491]
**Describe the bug**
The current implementation of our FocusPicker, TagPicker, DatePicker, etc. housed in the TaskCreator use `<span>` tags with the `role="presentation"` attribute, despite that they behave in the same manner and functionality as buttons. This does not affect the functionality, nor does it raise linter errors/warnings, however. This was noted first in #485. 

**To Check**
Check any one of the `_`Picker components in `src/components/TaskCreator/`.

**Comment by SamChou19815** [comment]
Fixed in #556.
